frontend/src/Herbs/test

```diff
@@ -0,0 +1,22 @@
+test('that component is deleted', (done) => {
```

Comment:
This seems to be a file without suffix. jest does not recognize this as a test. Either delete the file or change the name to execute the test.

Reply:
is deleted